paper-input.html

```diff
@@ -54,8 +54,12 @@
 
     behaviors: [
       Polymer.PaperInputBehavior
-    ]
+    ],
 
+    /** Re-fire change event that does not bubble across shadow roots. */
+    _changeAction: function() {
+      this.fire('change');
```

Comment:
@kevinpschaaf, is Polymer.Setting.useShadow ok for determining whether it's using native shadow DOM?